### PR TITLE
Lab/customizable select

### PR DESCRIPTION
**Pull request by tesk9**
For https://www.pivotaltracker.com/story/show/170709954

Adds Nri.Ui.Select.V7, which customizes the arrows used to indicate that this element is a dropdown. Replaces https://github.com/NoRedInk/noredink-ui/pull/119.

This PR also adjusts the border (adds error color, focus color, and increases bottom width) per "Inputs" guidelines in the zeplin styleguide.

Screen Shot 2020-01-16 at 1 16 06 PM

<details>
<summary>Kinda hard to see given focus outline, but the border is blue on focus.
</summary>

Screen Shot 2020-01-16 at 1 16 14 PM

</details>

cc @NoRedInk/design

### CODE REVIEW

**Comment by staceyadams** [comment]
🙌

**Comment by Arkham** [comment]
@tesk9 I couldn't find this branch on origin any longer, would you mind pushing again?

File: src/Nri/Ui/Select/V7.elm [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+    in
+    Css.batch
+        [ """<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="12px" height="16px" viewBox="0 0 12 16"><g fill=" """
+            ++ color
+            ++ """ "><path d="M2.10847,9.341803 C1.65347,8.886103 0.91427,8.886103 0.45857,9.341803 C0.23107,9.570003 0.11697,9.868203 0.11697,10.167103 C0.11697,10.465303 0.23107,10.763503 0.45857,10.991703 L5.12547,15.657903 C5.57977,16.114303 6.31897,16.114303 6.77537,15.657903 L11.44157,10.991703 C11.89727,10.536003 11.89727,9.797503 11.44157,9.341803 C10.98657,8.886103 10.24667,8.886103 9.79167,9.341803 L5.95007,13.182703 L2.10847,9.341803 Z"/><path d="M1.991556,6.658179 C1.536659,7.11394 0.797279,7.11394 0.3416911,6.658179 C0.1140698,6.43004 0,6.13173 0,5.83325 C0,5.53476 0.1140698,5.23645 0.3416911,5.00831 L5.008185,0.34182 C5.463081,-0.11394 6.202461,-0.11394 6.65805,0.34182 L11.32454,5.00831 C11.78031,5.4639 11.78031,6.202592 11.32454,6.658179 C10.86965,7.11394 10.13027,7.11394 9.674679,6.658179 L5.833118,2.81679 L1.991556,6.658179 Z"/></g></svg> """
+            |> urlUtf8
+            |> Css.property "background"
```
Comment (BrianHicks) [inline src/Nri/Ui/Select/V7.elm]: whoah, I had no idea you could directly embed like this 💥

File: src/Nri/Ui/Select/V7.elm [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+        , VendorPrefixed.property "appearance" "none"
+        , Css.backgroundRepeat Css.noRepeat
+        , Css.property "background-position" "center right 10px"
+        , Css.backgroundOrigin Css.contentBox
+        ]
+
+
+urlUtf8 : String -> String
+urlUtf8 content =
+    """url('data:image/svg+xml;utf8,""" ++ content ++ """')"""
```
Comment (BrianHicks) [inline src/Nri/Ui/Select/V7.elm]: love this li'l helper method ❤️

**Pull request merged**